modules/appeals_api/config/schemas/v2/10182.json

```diff
@@ -0,0 +1,150 @@
+// if/when changes are made to this file, please notify VSA claims & appeals team to make corresponding changes in
+// lib/decision_review/schemas/NOD_create_request_body_schema.json
+
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Part of the API Styleguide is to try and use the most recent schema version. Do we have a particular reason that you can recall that we're staying with draft 7? I feel like there may have been something w/ the va.gov frontend preferring certain schema layouts over others, but I don't recall that extended all the way out to our schema version or not...

Reply:
On second thought, this may be easier to tackle w/ v3. Still, something to keep in mind.